### PR TITLE
Implement a default home page selector

### PR DESCRIPTION
**Pull request by AliMickey**
https://github.com/photoprism/photoprism/issues/1354

I have never used this framework before, apologies if there are any obvious mistakes. 

Its a fairly simple PR, please do consider merging as this would benefit my use case at home. 

Thanks

### CODE REVIEW

**Comment by graciousgrey** [comment]
Thank you for your pull request, we will have a look after our vacation :)

File: frontend/src/options/options.js [hunk 1]
```diff
@@ -236,6 +236,70 @@ export const Themes = () => [
...
+    value: "favorites",
+    disabled: false,
+  },
+  {
+    text: $gettext("Moments"),
+    value: "moments",
+    disabled: false,
+  },
+  {
+    text: $gettext("Calender"),
```
Comment (graciousgrey) [inline frontend/src/options/options.js:267]: Needs to be Calendar

File: frontend/src/options/options.js [hunk 2]
```diff
@@ -236,6 +236,70 @@ export const Themes = () => [
...
+    value: "calender",
```
Comment (graciousgrey) [inline frontend/src/options/options.js:268]: calendar ;)

**Comment by graciousgrey** [comment]
Setting the default page works :)
What we need to ensure is, that a user cannot set a default page that is disabled.
E.g. if a user disables private there should be no private option in the default page selector

**Comment by AliMickey** [comment]
Sounds good,

I am unable to build for some reason, however I think that should be very simple. 

Here are screenshots of what I think will do the trick? I'm not sure if the settings features uri is correct. Please correct if wrong. 

Also, does the import have to be:

`import { config } from "../session";`

or

`import { config } from "session";`

image

**Comment by AliMickey** [comment]
Do you need me to do this on my side so you can review it? I sponsored the project so I could get this through, as well as the native app API fix that's required.

**Comment by lastzero** [comment]
Thank you very much! We can finish this if you're not in a hurry. Currently working on facial recognition. We have a small team, and doing our very best to get everything done as soon as possible.

**Comment by AliMickey** [comment]
Thanks for an update, just wanted to make sure the issues were not forgotten.

**Comment by platima** [comment]
Aaaah @AliMickey what are you doing?

**Comment by AliMickey** [comment]
Ha! I was cleaning up my repos. and assumed this was a remnant one. 

Demonstrates the state of this project unfortunately, moving over to Immich at the moment. 

To the devs who might see this, I really do understand the immense workload and effort required by the team (which is still only the original couple despite recurring income, forgive me if this has changed) for a project of this magnitude, but unfortunately progress has slowed down. 

I wish you and the team the best of luck with this, I appreciate the hard work and for giving the community an alternative to Google Photos etc.

**Comment by platima** [comment]
Hey @AliMickey looks like someone didn't like your comment, even if it did have a positive tone. Each to their own though and I've got the email notification of the comment if you ever need it.

Regarding your move though, I've gone the opposite way and moved from Immich to Photoprism purely for the tags/labels and the folder view, as Immich seems to have lost its tagging capability at some point, and has no folder view.

Just my personal preference really, but I have offered Theresa and Michael some funds to cover this change request!

**Comment by lastzero** [comment]
@platima Honestly, I'm not sure why this slipped through - probably, yes, because of all the other work we have going on. Based on your interest, I had planned to get it implemented this week.

Ultimately, these discussions cause a lot of noise because we get an email notification every time (just like you), which distracts us from our work and slows things down for everyone (exactly what has been criticized with a "positive tone", yet it's not helpful in any way). I'm not making this up to annoy you, it's written right here in our README.md:

- https://github.com/photoprism/photoprism?tab=readme-ov-file#github-issues-%EF%B8%8F

**So I'm going to move ahead and close this discussion. If you have questions or want to provide personal feedback, you can do so via email. Thank you both for your support!**

**Pull request closed without merging**